src/mugs.js

```diff
@@ -468,6 +468,20 @@ define([
         }
     });
 
+    Object.defineProperty(Mug.prototype, "hashtagPath", {
```

Comment:
The number of places where `mug.absolutePath` has been replaced with `mug.hashtagPath` is ... staggering. I'm wondering if it would have made more sense to change the implementation of `mug.absolutePath` to return what is now returned by `mug.hashtagPath`? Although on looking I see that there is still a lot of code that use `absolutePath`.

How does one know when to use each property? Is there a simple rule for that? Is it the case that `mug.hashtagPath` is now preferred and `mug.absolutePath` is deprecated? Whatever it is, it would be good to document it here.